bundles/binding/org.openhab.binding.bticino/src/main/java/org/openhab/binding/bticino/internal/BticinoBinding.java

```diff
@@ -44,7 +44,7 @@
      * <code>'^(.*?)\\.(host|port)$'</code>
      */
     private static final Pattern EXTRACT_BTICINO_GATEWAY_CONFIG_PATTERN = Pattern
-            .compile("^(.*?)\\.(host|port|rescan_secs)$");
+            .compile("^(.*?)\\.(host|port|passwd|rescan_secs)$");
```

Comment:
Should add an `@author` tag for yourself in the class' javadoc header.

Reply:
done